Tensile/LibraryIO.py

```diff
@@ -34,7 +34,7 @@
 try:
   import msgpack
 except ImportError:
-  printExit("You must install MessagePack for Python to use Tensile (to parse config files). See https://github.com/msgpack/msgpack-python for installation instructions.")
+  print("Message pack python library not detected. Must use YAML backend instead.")
```

Comment:
Instead of "Must use YAML..."
Might be good idea to have "Falling back to YAML..."